.tp-config.json

```
@@ -10,8 +10,11 @@
     "src/api/middlewares/*.*",
     "src/lib/config.js"
   ],
-  "generator": ">=0.41.0 <2.0.0",
+  "generator": ">=0.46.0 <2.0.0",
   "filters": [
     "@asyncapi/generator-filters"
-  ]
+  ],
+  "hooks": {
+    "@asyncapi/generator-hooks": "createAsyncapiFile"
```

Shouldn't it be like this?

```suggestion
    "@asyncapi/generator-hooks": ["createAsyncapiFile"]
```

---

@fmvilas I made sure both is supported, so in case of single hook name just string, if more, Array

---

Oh that's great :+1: